src/sentry/options/defaults.py

```diff
@@ -135,7 +135,7 @@
 register('vsts.client-secret', flags=FLAG_PRIORITIZE_DISK)
 
 # Snuba
-register('snuba.use_group_id_column', default=False)
+register('snuba.use_group_id_column', default=True)  # TODO: Set back!
```

Comment:
This option will actually be gone, or default `True` by the time I want to merge this.

Reply:
This is currently the cause of (unrelated) test failures. I'm going to fix those on another branch/PR now.